pkg/kubelet/pod_workers.go

```diff
@@ -60,11 +60,15 @@ type workUpdate struct {
 	// The pod state to reflect.
 	pod *api.BoundPod
 
+	// Whether there exists a mirror pod for pod.
+	hasMirrorPod bool
```

Comment:
I think that'd be slightly confusing. None of the pods we are syncing are actually mirror pods......but they may have mirror pods.